docs/tutorial/intro_to_zydeco.md

```diff
@@ -0,0 +1,168 @@
+# Intro to Zydeco
+## Types
+Zydeco has two main types: value type (`A`) and computation type (`B`). 
```

Comment:
two *kinds* of types

Reply:
Then I would also add something to think about what they mean: A value type classifies inert data and a computation type classifies programs that compute things